### ISSUE TITLE
Configuration not working

### ISSUE THREAD

**Issue by hbjcr** [bmcclure/ha-aquanta#1]
## Describe the bug
After adding the repository to HACS and restarting HA, "Aquanta" doesn't shows up in the integrations list

HA v2022.6.7

**Comment by bmcclure** [comment]
Thanks for the report!

Hmm, so far I've been unable to reproduce this issue. I've tried a fresh install of HA as well as my fully loaded real HA instance and it works for me in both.

For clarification, do you mean the repo doesn't show up in HACS after adding it, or you were able to add it download it through HACS but it doesn't show up when adding an integration in HA?

I'll keep trying to see if I can reproduce this somehow, but so far I'm not sure what's different between your setup and the ones I'm testing.

Do you see anything in the HA logs that could be relevant? Perhaps an HACS error if it's unable to pull the repository information?

Something I've run into is HACS running out of GitHub API requests and having to wait a while for it to pull any new information from GitHub. Just want to confirm it isn't something like that. Thanks!

**Comment by hbjcr** [comment]
I'm running HA from a VM. I'm using this .ova image: https://www.home-assistant.io/installation/alternative

I only see these 2 entries on my log: 
2022-07-01 23:52:58 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2022-07-01 23:52:58 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration aquanta which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant

Basically, if I go to HACS' integration module, I see Aquanta listed, it appears to be just fine. But if I go to Settings -> Devices & Integrations -> Integrations, and click on the "Add Integration" button and search for "Aquanta" it doesn't shows up. I believe my installation is fine because I do have other HACS integrations working well.

**Comment by hbjcr** [comment]
I tried rebooting my VM and the integration wasn't still there; using SSH I was able to confirm the files were showing up at the right location, but still nothing. Then I enabled an external URL for my instance and it's suddenly working without problems. I'm not sure if the external URL had anything to do with it or if it simply took longer to load it after my reboot, in any case, it appears to be working now. Thanks, it looks really well done.

**Comment by bmcclure** [comment]
That's great to hear, and thank you!

There are a few quirks I need to work out but it can read the temperature, the operating mode, and can set and unset away mode and those were the main things I needed so hopefully it's useful for others too!

**Comment by bmcclure** [comment]
Version 2 resolves several issues and adds new features. If you happen to still be having any issues, give that a try! And feel free to let me know if there are still problems.
